warrior/Warriorspace/Config_files/Samples/CLI_Test_Data_Sample.xml

```diff
@@ -413,6 +436,34 @@ Also refer the python regex documentation https://docs.python.org/2/library/re.h
 		<v7 found="yes" search="Commit complete"/>
 	</testdata>
 
+<!--
+		========================
+		II.d : multiple patterns
+		=========================
+			For storing multiple patterns from the response the tags should be provided as shown below:
+			:resp_keys:			(comma separated list of keys) for specifying the patterns
+			:resp_ref:			(comma separated list of vars) for storing the response corresponding to patterns
+			:inorder_resp_ref:	("yes" or "no" default "yes") If yes retrieves the patterns from the response in order
+								If no, retrieves the patterns from the response individually everytime
+
+-->
+	<global>
+		<keys>
+			<gk resp_pattern_req="No such file or directory"/>
+		</keys>
+	</global>
+    <command send="ls NonExistTitleRowDir"
+             end="\$" verify="v1" resp_keys="k1, k2, gk"
+             resp_ref="lsoutK1,lsoutK2, lsoutK3"
+             inorder_resp_ref="n"/>
+    <v1 found="yes" search="No such file or directory" />
+    <k1 resp_pattern_req="ls: "/>
+    <k2 resp_pattern_req="cannot access.*:"/>
+
+<!-- This would store "ls:" (ref k1) pattern/text in lsoutK1,
+    "cannot access.*:" (ref k2) pattern/text in lsoutK2 and
+    "No such file or directory" (ref gk from global section) pattern/text in lsoutK3
+-->
```

Comment:
I suggest to place II.d details in II.b section since the existing rep_req/resp_pat examples are provided under II.b section.